### PR TITLE
Fix a minor issue with README when we do the release

### PR DESCRIPTION
**Pull request by arunkumarchacko**
Removing the release tag for BQ connector. Will add it back when we automate the release for BQ connector

### CODE REVIEW

File: README-template.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ To add a dependency on one of the connectors using Maven, use the following:
     <dependency>
       <groupId>com.google.cloud.bigdataoss</groupId>
       <artifactId>bigquery-connector</artifactId>
-      <version>${next-bigquery-connector-release-tag}</version>
+      <version>hadoop2-1.2.0</version>
```
Comment (Deependra-Patel) [inline README-template.md:91]: Why are we hardcoding the value here?

File: README-template.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ To add a dependency on one of the connectors using Maven, use the following:
     <dependency>
       <groupId>com.google.cloud.bigdataoss</groupId>
       <artifactId>gcs-connector</artifactId>
-      <version>${next-gcs-connector-release-tag}</version>
+      <version>hadoop2-${next-gcs-connector-release-tag}</version>
```
Comment (Deependra-Patel) [inline README-template.md:81]: We also publish for hadoop3 right?

Comment (arunkumarchacko) [reply README-template.md:81]: Yes, we publish for both hadoop2 and hadoop3 from this branch.

File: README-template.md (hunk 1 above)
Comment (arunkumarchacko) [reply README-template.md:91]: This is the version for BQ connector. we have not released this in a while and probably will not. We have removed this from master for e.g. We can add this back if we plan to add automation for Hive BQ connector release.

**Review by Deependra-Patel — APPROVED** [review]

**Pull request merged**